docs/resources/file.md

```diff
@@ -0,0 +1,104 @@
+---
+subcategory: "Storage"
+---
+# databricks_file Resource
+
+This resource allows uploading and downloading files in [databricks_volume](volume.md) up to 5GiB in octet-stream.
+
+## Example Usage
+
+In order to manage a file on Unity Catalog Volumes with Terraform, you must specify the `source` attribute containing the full path to the file on the local filesystem.
+
+```hcl
+resource "databricks_catalog" "sandbox" {
+  metastore_id = databricks_metastore.this.id
+  name         = "sandbox"
+  comment      = "this catalog is managed by terraform"
+  properties = {
+    purpose = "testing"
+  }
+}
+
+resource "databricks_schema" "things" {
+  catalog_name = databricks_catalog.sandbox.name
+  name         = "things"
+  comment      = "this schema is managed by terraform"
+  properties = {
+    kind = "various"
+  }
+}
+
+resource "databricks_storage_credential" "external" {
+  name = "creds"
+  aws_iam_role {
+    role_arn = aws_iam_role.external_data_access.arn
+  }
+}
+
+resource "databricks_external_location" "some" {
+  name            = "external-location"
+  url             = "s3://${aws_s3_bucket.external.id}/some"
+  credential_name = databricks_storage_credential.external.name
+}
+
+resource "databricks_volume" "this" {
+  name             = "quickstart_volume"
+  catalog_name     = databricks_catalog.sandbox.name
+  schema_name      = databricks_schema.things.name
+  volume_type      = "EXTERNAL"
+  storage_location = databricks_external_location.some.url
```

Comment:
I would not use the term "external_location" here since it has a very specific meaning in Dtabricks, which is different from Volume

Reply:
external has to be used here. it references the previously created `databricks_external_location` resource, which is named `some`.